### PR TITLE
feat: dark theme

### PR DESCRIPTION
**Pull request by SiTaggart**
Add dark theme to Paste

- Deprecated old light / dark token naming
- Introduce weak / stron token replacements
- replace usage of light / dark design tokens in components
- create the dark theme in design tokens and add it to the theme package
- update storybook layout to display light and dark themes
  - allow chromatic to snapshot both themes for each story stacked

### CODE REVIEW

File: packages/paste-design-tokens/tokens/global/index.yml [hunk 1]
```diff
@@ -1,8 +1,11 @@
 imports:
+  - ./background-color-deprecated.yml
```
Comment (TheSisb) [inline packages/paste-design-tokens/tokens/global/index.yml:2]: Clean

File: packages/paste-theme/src/themes/dark/index.ts [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+  boxShadows,
+  sizings,
+  spacings,
+  textColors,
+  zIndices,
+} from '@twilio-paste/design-tokens/dist/themes/dark/tokens.es6';
+
+import {generateThemeFromTokens} from '../../generateThemeFromTokens';
+
+export const DarkTheme = generateThemeFromTokens(
```
Comment (TheSisb) [inline packages/paste-theme/src/themes/dark/index.ts:19]: Did we merge the thing that makes this function take an object instead?

**Review by TheSisb — APPROVED** [review]
Looks solid. Are we going to bump a major when we remove the deprecated tokens in the future? Or should the major prepatch happen here?

**Review by richbachman — APPROVED** [review]
Looks good to me. Nice work on the storybook side-by-side view!

**Comment by SiTaggart** [comment]
> Are we going to bump a major when we remove the deprecated tokens in the future? Or should the major prepatch happen here?

I think Major when we remove them from the package. I'm tempted to maybe make a codemod?

File: .changeset/silent-phones-bow.md [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+'@twilio-paste/button': patch
+'@twilio-paste/card': patch
+'@twilio-paste/checkbox': patch
+'@twilio-paste/combobox': patch
+'@twilio-paste/disclosure': patch
+'@twilio-paste/help-text': patch
+'@twilio-paste/input-box': patch
+'@twilio-paste/menu': patch
+'@twilio-paste/modal': patch
+'@twilio-paste/popover': patch
```
Comment (SiTaggart) [inline .changeset/silent-phones-bow.md:14]:
```suggestion
'@twilio-paste/popover': patch
'@twilio-paste/pagination': patch
```

**Pull request merged**